tslint.json

```diff
@@ -3,6 +3,7 @@
 		"build/lib/tslint"
 	],
 	"rules": {
+		"no-string-throw": true,
```

Comment:
I went manually through all occurrences of `throw` in the codebase and found 2 cases. Not sure about this one:
https://sourcegraph.com/github.com/Microsoft/vscode/-/blob/extensions/typescript/src/features/implementationsCodeLensProvider.ts#L45-46
@mattbierner